subcommand/inject-connect/command.go

```diff
@@ -131,6 +136,37 @@ func (c *Command) Run(args []string) int {
 		return 1
 	}
 
+	var cpuLimit, cpuRequest, memoryLimit, memoryRequest resource.Quantity
```

Comment:
Could move this out into its own function.

Reply:
I think it's fine.